modules/arrow/src/schema/convert-arrow-schema.ts

```diff
@@ -71,7 +71,7 @@ export function deserializeArrowMetadata(metadata?: SchemaMetadata): Map<string,
 export function serializeArrowField(field: ArrowField): Field {
   return {
     name: field.name,
-    type: serializeArrowType(field.type),
+    type: field.type.toString(),
```

Comment:
The test case failed because `serializeArrowType(field.type)` can't handle the class 'Int', which could be Int8/16/32/64 based on the `bitwise` value. 

It seems that the Type class in apache-arrow has `toString()` to get the stringified arrow field type. I am checking if it compatible with the `deserializeArrowType()` function.

Reply:
It might be easier in some places to use `arrow.Type.isUint8()` etc instead of manually checking the type yourself